common/jmeter/perf-test-is.sh

```diff
@@ -330,7 +330,7 @@ function run_test_data_scripts() {
 
     echo "Running test data setup scripts"
     echo "=========================================================================================="
-    declare -a scripts=("TestData_SCIM2_Add_User.jmx" "TestData_Add_OAuth_Apps.jmx" "TestData_Add_SAML_Apps.jmx")
+    declare -a scripts=("TestData_SCIM2_Add_User.jmx" "TestData_Add_OAuth_Apps.jmx" "TestData_Add_SAML_Apps.jmx" "Post_image.jmx")
```

Comment:
Can we rename the data script according to the other data adding scripts, please? Suggestion: **TestData_Add_images**